src/triton_interfaces/msg/PipelineType.msg

```diff
@@ -1,5 +1,6 @@
 # Available pipeline types
 string TYPE_EXAMPLE = "example"
+string TYPE_OBJECT_RECOGNITION = "object_recognition"
```

Comment:
Since the object recognition isn't itself a pipeline (but the node can be used as part of a pipeline, like the gate pipeline for example) it shouldn't have it's own pipeline type, I get you probably used this for testing but now that you have a launch file which can do the same thing you can remove this